index.html

```diff
@@ -231,5 +231,58 @@ <h2>About me</h2>
         </ul>
       </div>
     </section>
+    <!-- contact page -->
+    <section class="contact-page">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Good job setting up the contact form :+1: , Please use the forms' ` action attribute` value to the exact URL which contains your form id, you can generate the form id by signing up and making an account on [formspree service](https://formspree.io/html). This is because we need to integrate our contact form with formspree so that users are able to contact us through our portfolio :pray: 

currently I am not able to contact you or send a message using the contact form please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200624762-9af333e0-40fa-429c-a418-5aec0dcc0190.png)
It actually tells us in the response that we have to check our hash id, so instead of {form_id} in your action attribute, please replace it with the actual form id :+1: